Cargo.toml

```diff
@@ -1,3 +1,3 @@
 [workspace]
-members = ["src/public"]
+members = ["src/public", "src/gpt_answer_service"]
```

Comment:
Why changing this?

Reply:
Because i want to build all features with command `cargo build --release --all`